.gitignore

```diff
@@ -3,3 +3,4 @@
 dist-newstyle
 stack.yaml
 cabal.project.local
+.ghci
```

Comment:
I’ve started using `cabal exec ghci` more and so I’m using this to set stuff up for that, but I don’t think we want to commit it since others will likely be using `cabal repl` and it might break or confuse things?